cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -112,7 +113,15 @@ def vm_analysis_provisioning_data(provider, analysis_type):
     vma_data = provider.data.vm_analysis_new
     provisioning_data = vma_data.provisioning
 
-    if not isinstance(provider, CloudProvider):
+    if provider.type == 'azure':
+        provisioning_data.setdefault('instance_type', vma_data.provisioning.instance_type)
+        provisioning_data.setdefault('virtual_net',
```

Comment:
I'd recommend using `cloud_network` - it has all required values - split it. virtual_net is used in wrapanapi/REST